clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/userdata/UserDataProviderAggregatorSpec.groovy

```diff
@@ -0,0 +1,105 @@
+package com.netflix.spinnaker.clouddriver.aws.deploy.userdata
+
+import spock.lang.Specification
+
+class UserDataProviderAggregatorSpec extends Specification {
```

Comment:
nit: Would prefer to see the base64-encoded test inputs as test resources instead and then base64 the file contents inside the test class, that way if the test inputs need to change, it'll be more readily obvious what specific part changed. Further, we won't need to worry about these comments drifting from the actual values.